desktop/src/main.rs

```diff
@@ -1006,14 +1006,18 @@ fn shutdown() {
     }
 }
 
-fn main() -> Result<(), Error> {
+fn main() {
     init();
     let opt = Opt::parse();
     let result = if opt.timedemo {
         run_timedemo(opt)
     } else {
         App::new(opt).map(|app| app.run())
     };
+    if let Err(ref error) = result {
+        eprintln!("\n\n{:?}", error);
+        shutdown();
```

Comment:
I think the `shutdown(); std::process::exit(1);` can be removed as it'll anyway happen right after the `if` (returning an `Error` from `main` causes the process to exit with a non-zero code). This effectively leaves just (with some modifications):
```rs
if let Err(error) = &result {
    eprintln!("{:?}", error);
}
```
And the rest can remain unchanged.

Reply:
Hi, since returning an anyhow::Error from main causes it to be printed, the error is printed twice on a non-windows OS. That was why I had set main to not return anything and just had it print once, but not sure if there is a better way to achieve the same result.

Reply:
Just remembered there is a better way, cleaned up main and got rid of those unnecessary lines.